pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: growthbook_sdk_flutter
 description: An open-source feature flagging and experimentation platform that makes it simple to alter features and execute A/B testing.
-version: 2.1.0+0
+version: 2.1.1+0
```

Comment:
As it updates the Dio version to the major we should consider updating the SDK version to the major.

Reply:
Done.